main/src/components/OutageMap.js

```diff
@@ -9,6 +9,24 @@ function OutageIndicator({ outage }) {
   const [coords, setCoords] = useState();
   //localStorage.clear();
   //console.log(JSON.parse(localStorage.getItem("user")));
+  const localUser = localStorage.getItem("user");
+  const userEmail = '"' + outage.user_email + '"';
+  // localUser user has "" around it so "" is added around outage.user_email so that it would satisfy the condition on line 40-66
+  if(localUser === userEmail) { 
+    console.log("user exist");
+  }
+  else {
+    console.log("User does not exist");
+  }
+
+  const [isLoggedIn, setIsLoggedIn] = useState(localUser === userEmail); //localStorage.getItem("user") === outage.user_email
+  function closeReport() { 
+    console.log("Closing Report");
+  }
+
+  function extendReport(){
+    console.log("Extending Report");
+  }
 
```

Comment:
These functions will need to make the calls to the backend to close and extend an outage. Other than that, looks good